Jantimizer/ExperimentSuite/experiments.json

```diff
@@ -1,22 +1,28 @@
 ﻿{
   "Experiments": [
     {
-      "ExperimentName": "Experiment One",
+      "ExperimentName": "Experiment One - Base EquiDepth",
       "RunParallel": true,
       "RunExperiment": true,
       "PreRunData": [
         {
           "ConnectorName": "POSGRESQL",
           "ConnectorID": "EquiDepth",
           "TestFiles": [
-            "BasicTests_SetupTables"
+            "BasicTests_SetupTables",
+            "WorldHappiness_With_GDP_Setup",
+            "Census_2013_Setup",
+            "COVID_Funding_Setup"
```

Comment:
I guess you just stole the issue i was working on :'(

Reply:
Also i still think its a better idea to wait with merging all these, until the optimiser can actually *work* with them.